backend/src/main/java/com/metamong/mt/domain/reservation/controller/ReservationController.java

```diff
@@ -31,4 +44,21 @@ public ResponseEntity<?> findReservationByRvtId(@PathVariable Long reservationId
         return ResponseEntity.ok(BaseResponse.of(reservationService.findReservationByRvtId(reservationId),
                 HttpStatus.OK, "예약 상세 정보 불러오기 성공"));
     }
+
+    @PostMapping("/recommends/{memberId}")
+    public Mono<RecommendationResponseDto> getRecommendations(@PathVariable int memberId)
+            throws JsonProcessingException {
+        List<ReservationInfoResponseDto> rvtInfo = reservationService.getTotalCount();
+
+        Map<String, Object> rvtInfoList = new HashMap<>();
+        rvtInfoList.put("reservation_info", rvtInfo);
+
+        ObjectMapper objectMapper = new ObjectMapper();
+        log.info("Sending request with body: " + objectMapper.writeValueAsString(rvtInfoList));
```

Comment:
이 부분은 디버깅을 위해 있는 부분 같습니다. 로그 레벨이 debug 혹은 trace일 때만 실행되도록 하면 좋을 듯해요.
```suggestion
        if (log.isDebugEnabled()) {
            log.info("Sending request with body: " + objectMapper.writeValueAsString(rvtInfoList));
        }
```